dso/model.py

```diff
@@ -17,3 +17,27 @@ class ScanArtifact:
         cm.ResourceAccess,
     ]
     label: dso.labels.ScanLabelValue
+
+
+@dataclasses.dataclass(frozen=True)
+class DependabotCoverageReportRepo:
+    repo: str
+    dependabot: bool
+
+
+@dataclasses.dataclass(frozen=False)
+class DependabotCoverageReport:
+    coverage: float
+    github: str
+    details: typing.List[DependabotCoverageReportRepo]
+
+    def calculate_overall_percentage(self):
+        t = 0
```

Comment:
stylistic nit: this is a very cheap operation. rather either already do it in the c'tor, or (pbly better) do the calculation upfront, and pass-in the calculated values (-> only use this dataclass as a "struct"). I think it is a bad pattern to have a class-definition leading to objects that are useless w/o calling a special method once